app/src/main/java/de/westnordost/streetcomplete/quests/barrier_type/AddBarrierTypeForm.kt

```diff
@@ -0,0 +1,40 @@
+package de.westnordost.streetcomplete.quests.barrier_type
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.quests.AImageListQuestAnswerFragment
+import de.westnordost.streetcomplete.quests.bike_parking_type.BarrierType.*
```

Comment:
```suggestion
import de.westnordost.streetcomplete.quests.barrier_type.BarrierType.*
```

Reply:
Thanks! I already commited it (not pushed), together with one more reference.